packages/fxa-payments-server/src/routes/Subscriptions/Reactivate/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { formatCurrencyInCents } from '../../../lib/formats';
+import DialogMessage from '../../../components/DialogMessage';
+import fpnImage from '../../../images/fpn';
+import { Plan, Customer } from '../../../store/types';
+import { metadataFromPlan } from '../../../store/utils';
+
+export default ({
+  onDismiss,
+  onConfirm,
+  plan,
+  customer,
+  periodEndDate,
+}: {
+  onDismiss: Function;
+  onConfirm: () => void;
+  plan: Plan;
+  customer: Customer;
+  periodEndDate: string;
+}) => {
+  const { webIconURL } = metadataFromPlan(plan);
+  const { last4 } = customer;
+
+  return (
+    <DialogMessage onDismiss={onDismiss}>
+      <img
```

Comment:
Where the product image gets used in the subscription redirect - visible in local storybook here after clicking "Resubscribe":

http://localhost:6006/?path=/story/routes-subscriptions--reactivation